### PR TITLE
Add `code_of_conduct`

### PR DESCRIPTION
**Pull request by pingelit**
This PR adds the Contributor Covenant code of conduct to the gallery.

### CODE REVIEW

**Review by f-brinkmann — APPROVED** [review]
Thanks - I would only suggest to replace [INSERT CONTACT METHOD] with `mailto:info@pyfar.org' :)

**Comment by pingelit** [comment]
Oh yes, the one thing one has to change and I miss it 🤦

**Review by mberz — APPROVED** [review]

**Review by f-brinkmann — CHANGES_REQUESTED** [review]
Wew could add a link to the contrubuting guidelines in the box *Add code, documentation, or funding*

**Review by f-brinkmann — APPROVED** [review]

**Comment by pingelit** [comment]
The more I think about it, both tiles should then include a link. So both _Add code, documentation, or funding_ as well as _Feedback, bugs, feature requests_. What do you think?

File: docs/community/index.rst [hunk 1]
```diff
@@ -29,9 +30,17 @@ community for the acoustics research community. We welcome any contributions
       are the issues on GitHub. The GitHub projects are linked through the
       Code buttons `pyfar.org <https://pyfar.org>`_. If you have own ideas for contributions, may it be
       code, or other things, it would be best to get in touch through `Slack <https://join.slack.com/t/pyfar/shared_invite/zt-2eacdhww2-iUiPnh_wuqg2zD939wL4kw>`_
-      or info@pyfar.org. Before starting, please have a look at the guidelines below.
+      or info@pyfar.org. Before starting, please have a look at the guidelines and our `code of conduct`_.
```
Comment (ahms5) [inline docs/community/index.rst:33]:
```suggestion
      or info@pyfar.org. Before starting, please have a look at the `guidelines <contributing.rst>`_ and our `code of conduct`_.
```
maybe like this?

File: docs/community/index.rst [hunk 2]
```diff
@@ -5,7 +5,8 @@ Community
 Pyfar is completely open source and it’s license allows unrestricted
 educational and commercial use. It is made by the acoustics research
 community for the acoustics research community. We welcome any contributions
-– go ahead if you are in the mood.
+– go ahead if you are in the mood. Please also note our
+`code of conduct`_.
```
Comment (ahms5) [inline docs/community/index.rst:9]:
```suggestion
– go ahead if you are in the mood.
```
this might be to much, its already mentiond below.

**Review by ahms5 — COMMENTED** [review]
thank you, just small suggetions.

**Comment by f-brinkmann** [comment]
+1 for linking contributing guidelines and code of conduct directly from the text. But maybe only once.

File: docs/community/index.rst (hunk 2 above)
Comment (mberz) [reply docs/community/index.rst:9]: agreed

**Review by mberz — CHANGES_REQUESTED** [review]
The code of conduct is still linked twice. I think we agreed that once does suffice.

**Comment by pingelit** [comment]
Thanks for the input. I removed the second link and kept the link outside the cards. This makes the most sense for me as it applies to both cards.

Regarding Ahms5's suggestion of linking to the guidelines, I skipped this as I thought it was outside the scope of this PR.

File: docs/community/index.rst [hunk 3]
```diff
@@ -5,7 +5,8 @@ Community
 Pyfar is completely open source and it’s license allows unrestricted
 educational and commercial use. It is made by the acoustics research
 community for the acoustics research community. We welcome any contributions
-– go ahead if you are in the mood.
+– go ahead if you are in the mood. For this, please note and follow our
+`code of conduct`_.
```
Comment (ahms5) [inline docs/community/index.rst:9]: Last meeting we decided to do it in this pr, we discussed to remove the link Here and add both links in the box as it was proposed by previous suggestions.

**Review by ahms5 — CHANGES_REQUESTED** [review]

**Pull request merged**